package.json

```diff
@@ -74,10 +74,8 @@
     "temp": "^0.8.3",
     "webpack": "^2.1.0-beta.25"
   },
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/yarnpkg/yarn.git"
-  },
+  "repository": "yarnpkg/yarn",
+  "bugs": "https://github.com/yarnpkg/yarn/issues",
```

Comment:
Yeah. This doesn't add anything that can't already be determined without it being present.